src/javascript/app/pages/user/accounts.js

```diff
@@ -38,7 +38,8 @@ const Accounts = (() => {
             // ask client to set residence first since cannot wait landing_company otherwise
             BinaryPjax.load(urlFor('user/settings/detailsws'));
         }
-        SetCurrency.cleanupPopup();
+        clearPopup();
+        window.addEventListener('beforeunload', clearPopup);
```

Comment:
why do we need before unload as well as on unload?

Reply:
When user changes url directly in browser url bar, we face the error on this card. I figured out onUnload function doesn’t work in this situation so I used  "beforeunload" event. Also "beforeunload" doesn’t work when user clicks back button of the browser (I've tested on Chrome).
So I need to use both of them to cover the error.

Reply:
@amirghadimi-fs then I suggest we do this in the common page loader script to always set the onuload script (if any) of the current page to the beforeunload event, because if it happens here it can happen on other pages too